ethsigner/core/src/integration-test/java/tech/pegasys/ethsigner/jsonrpcproxy/SigningEthSendTransactionIntegrationTest.java

```diff
@@ -155,9 +173,9 @@ void signTransactionWhenEmptyReceiverAddress() {
   @Test
   void signTransactionWhenEmpty0xReceiverAddress() {
     final Request<?, EthSendTransaction> sendTransactionRequest =
-        sendTransaction.withReceiver("0x");
+        sendTransaction.request(transactionBuilder.withTo("0x"));
```

Comment:
`withTo` does that refers to receiver ??

Reply:
It does, the field is to but in our DSL at transaction level we were referring to as receiver to make it easier to understand. I'm thinking we just change sender -> from and receiver -> to to be consistent with the fields.